app/main/views/users.py

```diff
@@ -0,0 +1,81 @@
+from datetime import datetime
+
+from flask import jsonify, request, abort
+from app.main import main
+from app.models import User
+from app import db, encryption
+from sqlalchemy.exc import IntegrityError
+from app.main import helpers
+from app.validation import validate_user_json_or_400, \
+    validate_user_auth_json_or_400
+
+
+@main.route('/users/auth', methods=['POST'])
+def auth_user():
+    json_payload = get_json_from_request('auth_users')
+
+    validate_user_auth_json_or_400(json_payload)
+
+    user = User.query.filter(
+        User.email_address == json_payload['email_address'].lower()).first()
+
+    if user is None:
```

Comment:
2 types of response - user dosn't exist and user gave incorrect password, returned so users of API can perform auditing/reporting etc of issues as per their own use case.